provider/equinix/environ.go

```diff
@@ -642,7 +642,8 @@ func validPlan(plan packngo.Plan, region string) bool {
 }
 
 func parseMemValue(v string) (uint64, error) {
-	var scaler = uint64(1)
+	// This is here to convert from GB (what Equini uses) to MB (what juju requires)
+	var scaler = uint64(1000)
```

Comment:
If it's always reported as GB why not set the scaler to 1024 and remove L648?

Reply:
Actually the API seems to return memory values as "xGB" whereas this func is expected to return the value in bytes. I think you should leave the initial scaler to 1 and change L648 to `scaler = 1073741824`